test/Serilog.Extensions.Logging.Tests/SerilogLoggerScopeTests.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Is the .NET Foundation copyright header intended here, and in the other test file below?

Reply:
I don't know what it needs to be so just copied it from another file.

I've had another look and there's a mix of files with this header, `Copyright [year] Serilog Contributors`, and no header. What should I change this and the other file to?

Reply:
Standard header as per serilog core without a year in all source files
Optional in test files
(I've yet to do the PR to make it completely consistent in core)
https://github.com/serilog/serilog/pull/1969#issuecomment-1782030648